nginx/domains/firmware.ffmuc.net.conf

```diff
@@ -77,11 +78,12 @@ server {
 
         # Add Gluon releases here that allow an upgrade path to the latest version
         # https://github.com/freifunk-gluon/gluon/wiki/Release-life-cycle#migration-paths-diagram
-        set $gluon_v2019_1_x gluon_v2021.1.x; # we do not have an upgrade path for v2018 and earlier yet
-        set $gluon_v2021_1_x gluon_v2021.1.x;
-        set $gluon_v2022_1_x gluon_v2023.1.x;
-        set $gluon_v2023_1_x gluon_v2023.1.x;
-        set $gluon_v2023_2_x gluon_v2023.2.x;
+        set $gluon_v2019_1_x gluon_v2021.1.x; # we want routers to upgrade from v2019 to v2021 and only then attempt v2023
+        set $gluon_v2021_1_x gluon_v2021.1.x_v2023.1.x;
+        set $gluon_v2022_1_x gluon_v2023.1.x_v2023.2.x;
+        set $gluon_v2023_1_x gluon_v2023.1.x_v2023.2.x;
+        set $gluon_v2023_2_x gluon_v2023.1.x_v2023.2.x;
+        set $gluon_next gluon_next;
```

Comment:
Note: we need to add some entries for gluon_next in the `TBA` section above once we have a real next release again. (cc @T0biii )